admin-cli/README-cli.md

```diff
@@ -0,0 +1,52 @@
+# admin-cli
+
+This is a command line tool for catalog admins to examine the contents of the catalog
```

Comment:
Wording nit - when I think of "examine", my mind goes to read-only operations.  I know that all of the currently supported commands fall in this camp but you might expand in the future to support options that modify the DB.

Reply:
Good point, I'll change the wording, and looking at the README again I think I'll edit it a bit overall.